core-parser/src/main/java/org/uniprot/core/parser/tsv/uniprot/comment/CatalyticActivityMap.java

```diff
@@ -56,8 +56,12 @@ private String extractRheaIds(List<CatalyticActivityComment> caComments) {
                 .filter(Reaction::hasReactionCrossReferences)
                 .map(Reaction::getReactionCrossReferences)
                 .flatMap(Collection::stream)
-                .filter(rr -> ReactionDatabase.RHEA.equals(rr.getDatabase()))
+                .filter(rr -> filterOutRheaCompIds(rr))
                 .map(CrossReference::getId)
```

Comment:
Minor comment:
Use method reference instead of lambda. It makes the code more neat.
this::filterOutRheaCompIds